pages/screens/Login/styles.module.css

```diff
@@ -0,0 +1,68 @@
+.container {
+  min-height: 100vh;
+  padding: 0 0.5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.main {
+  align-items: center;
+  padding: 4rem 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+}
+
+.form {
+  align-self: flex-start;
+  display: flex;
+  flex-direction: column;
+  padding-top: 2rem;
+}
+
+.label {
+  margin-bottom: 0.8rem;
+}
+
+.inputEmail {
+  margin-bottom: 2rem;
+}
+
+.inputPassword {
+  margin-bottom: 6rem;
+}
+
+.button {
+  align-self: center;
+}
+
+.footer {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  padding-bottom: 2rem;
+  justify-content: flex-end;
+}
+
+@media screen and (min-width: 400px) {
+  .main {
+    max-width: 60rem;
+    padding: 4rem 2rem;
+    width: 80%;
```

Comment:
use padding on the parent instead of this

Reply:
Actually i don t need width or padding, it didn t work before,